### PR TITLE
Update Ecosystem.md

### PR DESCRIPTION
**Pull request by chapuletta**
Added 2 plugins developed by me:
Fastify Swagger UI: https://github.com/chapuletta/fastify-swagger-ui
Fastify KnexJS: https://github.com/chapuletta/fastify-knexjs

### CODE REVIEW

**Comment by mattbrunetti** [comment]
Thanks @chapuletta .. Can't wait to try out fastify-knexjs for sure! Might try fastify-swagger-ui as well!

**Comment by delvedor** [comment]
Hello!
Why not directly contribute to [fastify-swagger](https://github.com/fastify/fastify-swagger) instead of copy pasting the implementation and declare it as "developed by me"? :)

**Comment by chapuletta** [comment]
@delvedor because i didn't know if the plugin must have the UI, maybe someone only want the yaml documentation. I use the implementation of the fastify-swagger plugin where i mention it, and put de UI.
If you think is better put the UI in the fastify-swagger plugin tell me and i do a pull request.

Thxs for advance!

**Comment by delvedor** [comment]
@chapuletta [fastify-swagger](https://github.com/fastify/fastify-swagger) do have the UI support.
In any case in the open source world is not nice copy paste code written by others and declare it as yours, even if the license allows it.
The next time a fork will be more appropriate.

If you want to contribute to [fastify-swagger](https://github.com/fastify/fastify-swagger) go ahead!
Regarding this pr please remove `fastify-swagger-ui` from the list.

**Comment by chapuletta** [comment]
@delvedor i know that the fastify-swagger dont have the UI, i use the code from de plugin for documentation generation and implement de UI i, mentioned that i used the code from de fastify-swagger but is not my intention say that the plugin is mine. My intention was not include de UI in the fastify-swagger plugin because im not the owner, for this reason i made other one where i "use" the fastify-swagger code where i mention it in the README.md.

Already remove the plugin from the list, and i gonna fork the fastify-swagger for UI support in a pull request.

**Comment by delvedor** [comment]
@chapuletta [fastify-swagger](https://github.com/fastify/fastify-swagger) **do have** the UI support :)
You can see how use it [here](https://github.com/fastify/fastify-swagger#swagger-options).

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -44,3 +44,4 @@ Plugins maintainted by the fastify team are listed under [Core](#core) while plu
 - [`fastify-session`](https://github.com/SerayaEryn/fastify-session) a session plugin for Fastify.
 - [`fastify-sse`](https://github.com/lolo32/fastify-sse) to provide Server-Sent Events with `reply.sse( … )` to Fastify.
 - [`fastify-ws`](https://github.com/gj/fastify-ws) WebSocket integration for Fastify — with support for WebSocket lifecycle hooks instead of a single handler function. Built upon [ws](https://github.com/websockets/ws) and [uws](https://github.com/uNetworking/bindings/tree/master/nodejs).
+- [`fastify-knexjs`](https://github.com/chapuletta/fastify-knexjs) Fastify plugin for support KnexJS Query Builder.
```
Comment (delvedor) [inline docs/Ecosystem.md]: Please follow the alphabetical order of the list :)

**Comment by chapuletta** [comment]
@delvedor but not have the UI inside, that was i did in the new plugin. Only copy the implementation of fastify-swagger for remove /, * and /docuementation endpoints for de UI. but if you want i remove and we are clear.

**Pull request merged**